tensorflow_datasets/datasets/amazon_us_reviews/amazon_us_reviews_dataset_builder.py

```diff
@@ -126,30 +96,22 @@ def _info(self):
     return self.dataset_info_from_configs(
         features=tfds.features.FeaturesDict(
             {
-                "data": collections.OrderedDict([
-                    ("marketplace", np.str_),
-                    ("customer_id", np.str_),
-                    ("review_id", np.str_),
-                    ("product_id", np.str_),
-                    ("product_parent", np.str_),
-                    ("product_title", np.str_),
-                    ("product_category", np.str_),
-                    ("star_rating", np.int32),
-                    ("helpful_votes", np.int32),
-                    ("total_votes", np.int32),
-                    ("vine", tfds.features.ClassLabel(names=["Y", "N"])),
-                    (
-                        "verified_purchase",
-                        tfds.features.ClassLabel(names=["Y", "N"]),
-                    ),
-                    ("review_headline", np.str_),
-                    ("review_body", np.str_),
-                    ("review_date", np.str_),
-                ])
+                "unixReviewTime": np.int32,
+                "reviewTime": np.str_,
+                "reviewerID": np.str_,
+                "reviewerName": np.str_,
+                "asin": np.str_,
+                "overall": np.int32,
+                "summary": np.str_,
+                "reviewText": np.str_,
+                "verified": np.bool_, # tfds.features.ClassLabel(names=["Y", "N"])
```

Comment:
Can you remove all the comments?

Reply:
Sure.
What about the comments on the top of the file?
```
# reviews (883,636 reviews)     metadata (186,637 products)
```
Should I remove them too?

Reply:
No, LGTM.